example/client/gateway/filter/README.md

```diff
@@ -0,0 +1,73 @@
+# How to use this example
+After launching the k8s cluster, do the following steps at the root directory.
+
+1. Deploy Vald cluster with filter-gateway
+```bash
+vim example/helm/values.yaml
+---
+...
+gateway:
+...
+    filter:
+        enabled: true
+...
+agent:
+    ngt:
+        dimension: 784
+        distance_type: l2
+...
+
+# deploy vald cluster
+helm repo add vald https://vald.vdaas.org/charts
+helm install vald vald/vald --values example/helm/values.yaml
+```
+
+2. Build and publish example ingress filter and egress filter docker image
+
+```bash
+# login to docker if needed, and setup your DockerHub ID
+docker login
+export DOCKERHUB_ID=<DOCKERHUB_ID>
+
+# build and publish ingress filter image
+docker build \
+    -f example/manifest/filter/ingress/Dockerfile \
+    -t $DOCKERHUB_ID/vald-ingress-filter:latest . \
+    --build-arg GO_VERSION=$(make version/go)
+
+docker push ${DOCKERHUB_ID}/vald-ingress-filter:latest
+
+# build and publish egress filter image
+docker build \
+    -f example/manifest/filter/egress/Dockerfile \
+    -t $DOCKERHUB_ID/vald-egress-filter:latest . \
+    --build-arg GO_VERSION=$(make version/go)
+
+docker push ${DOCKERHUB_ID}/vald-egress-filter:latest
+```
```

Comment:
```suggestion
    ```bash
    # login to docker if needed, and setup your DockerHub ID
    docker login
    export DOCKERHUB_ID=<DOCKERHUB_ID>
    
    # build and publish ingress filter image
    docker build \
        -f example/manifest/filter/ingress/Dockerfile \
        -t $DOCKERHUB_ID/vald-ingress-filter:latest . \
        --build-arg GO_VERSION=$(make version/go)
    
    docker push ${DOCKERHUB_ID}/vald-ingress-filter:latest
    
    # build and publish egress filter image
    docker build \
        -f example/manifest/filter/egress/Dockerfile \
        -t $DOCKERHUB_ID/vald-egress-filter:latest . \
        --build-arg GO_VERSION=$(make version/go)
    
    docker push ${DOCKERHUB_ID}/vald-egress-filter:latest
    ```
```